src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -755,6 +755,37 @@ GUI_TEST_CLASS_DEFINITION(test_7384_2) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7401) {
+    // 1. Open human_T1.fa.
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+
+    // 2. Select any part of sequence.
+    auto panView = GTUtilsSequenceView::getPanViewByNumber(os);
+    CHECK_SET_ERR(panView != nullptr, "No pan view found!");
+
+    auto startPoint = panView->mapToGlobal(panView->rect().center());
+    auto endPoint = QPoint(startPoint.x() + 150, startPoint.y());
+    GTMouseDriver::moveTo(startPoint);
```

Comment:
Can we use dragAndDrop() utility here? It is not as trivial as move/press/move/release

Reply:
Well, yes, replaced